spec/controllers/users/confirmations_controller_spec.rb

```diff
@@ -9,7 +9,7 @@
     it 'tracks nil email confirmation token' do
       analytics_hash = {
         success: false,
-        error: "Confirmation token can't be blank",
+        error: 'Confirmation token Please fill in this field.',
```

Comment:
Yeah, sorry I meant to try and clean this one up.  The rails default for blank input fields is "can't be blank", so I updated errors.messages.blank to "Please fill in this field."

Before:
![image 2016-11-17 at 2 23 50 pm](https://cloud.githubusercontent.com/assets/1178494/20404210/f0fe0694-acd1-11e6-8b5a-76fbf8396d56.png)

After:
![image 2016-11-17 at 2 22 08 pm](https://cloud.githubusercontent.com/assets/1178494/20404219/fd74b06c-acd1-11e6-867d-ea45c7ce8998.png)

Because of the way our forms are laid this was the only way I could see to format them properly, but since this specific error above also uses that translation string it doesn't read quite properly.  The error is actually never displayed to the user on the frontend however, it uses a flash message instead.  Not sure if there is a way to substitute that flash message in instead?